artifactory/commands/transferfiles/transferfileprogress.go

```diff
@@ -170,6 +170,10 @@ func (t *TransferProgressMng) AddPhase2(tasksPhase2 int64) {
 	t.phases = append(t.phases, t.barsMng.NewTasksWithHeadlineProg(tasksPhase2, "Phase 2: Transferring newly created and modified files", false, progressbar.GREEN, TimeSlots.String()))
 }
 
+func (t *TransferProgressMng) AddPhase3(tasksPhase3 int64) {
+	t.phases = append(t.phases, t.barsMng.NewTasksWithHeadlineProg(tasksPhase3, "Phase 3: Retry transferring all files that had upload errors", false, progressbar.GREEN, Files.String()))
```

Comment:
Retry transferring all files that had upload errors
-->
Retry transfer failures